src/Cafe/OS/libs/nn_fp/nn_fp.cpp

```diff
@@ -607,6 +623,20 @@ namespace nn
 			return resultBuf != 0 ? 1 : 0;
 		}
 
+		nnResult UpdateCommentAsync(uint16be* newComment, void* funcPtr, void* customParam)
+		{
+			FP_API_BASE();
+			auto ipcCtx = std::make_unique<FPIpcContext>(iosu::fpd::FPD_REQUEST_ID::UpdateCommentAsync);
+			uint32 commentLen = CafeStringHelpers::Length(newComment, 17);
+			if (commentLen >= 17)
```

Comment:
Replacing 17 with `MY_COMMENT_LENGTH+1` makes it clearer where the size comes from

Reply:
Actually nvm, MY_COMMENT_LENGTH is 18 and not 16. So it's using a different size as the base